lib/SPIRV/libSPIRV/SPIRVOpCode.h

```diff
@@ -71,7 +71,7 @@ inline bool isAtomicOpCode(Op OpCode) {
 }
 inline bool isBinaryOpCode(Op OpCode) {
   return ((unsigned)OpCode >= OpIAdd && (unsigned)OpCode <= OpFMod) ||
```

Comment:
I am wondering why this cannot be: (unsigned)OpCode >= OpIAdd && (unsigned)OpCode <= OpSMulExtended)
Ref: https://github.com/KhronosGroup/SPIRV-Headers/blob/79743b899fde5c954897b2694291002626358fac/include/spirv/unified1/spirv.h#L1377

Reply:
The problem here is that I'm not sure what this function name actually means.
From context we can assume that binary in this case means two registers operator. If this is true than it means that this function doesn't cover all possible ops like e.g OpFOrdNotEqual.